common/components/Footer/index.tsx

```diff
@@ -220,36 +227,49 @@ export default class Footer extends React.Component<Props, State> {
                 OTHER DEALINGS IN THE SOFTWARE.
               </b>
             </Modal>
-
+            <p>Latest Block#: {this.props.latestBlock}</p>
             <p>&copy; 2017 MyEtherWallet, LLC</p>
           </div>
 
-          <div className="Footer-column Footer-info">
+          <div className="Footer-info">
             <h5>
               <i aria-hidden="true">👫</i>
               You can support us by supporting our blockchain-family.
             </h5>
-            <p>Consider using our affiliate links to...</p>
-            <ul>
-              {LINKS_SUPPORT.map(link => {
-                return (
-                  <li key={link.href}>
-                    <a href={link.href} target="_blank">
-                      {link.text}
-                    </a>
-                  </li>
-                );
-              })}
+            <p>Consider using our affiliate links to</p>
+            <ul className="Footer-affiliate-wrap">
+              <AffiliateTag
+                link={bityReferralURL}
+                text="Swap ETH/BTC/EUR/CHF via Bity.com"
+              />
+            </ul>
+            <p>Buy a</p>
+            <ul className="Footer-affiliate-wrap">
+              <AffiliateTag
+                link="https://www.ledgerwallet.com/r/fa4b?path=/products/"
+                text="Ledger Nano S"
+              />
+              <AffiliateTag
+                link="https://trezor.io/?a=myetherwallet.com"
+                text="TREZOR"
+              />
+              <AffiliateTag
+                link="https://digitalbitbox.com/?ref=mew"
+                text="Digital Bitbox"
+              />
             </ul>
-
             <h5>
               <i aria-hidden="true">💝</i>
               {translate('FOOTER_2')}
             </h5>
             <ul>
               <li>
-                {' '}
-                ETH: <span className="mono wrap">{donationAddressMap.ETH}</span>
+                ETH: mewtopia.eth{' '}
+                <span className="mono wrap">
+                  <a href="https://etherscan.io/address/0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8">
+                    {donationAddressMap.ETH}
+                  </a>
```

Comment:
This should also be a NewTabLink, we probably don't want them to lose state for clicking on this